accounts/forms.py

```diff
@@ -44,3 +48,20 @@ def signup(self, request, user):
             user.is_active = False
 
         user.save()
+
+
+class EditProfileForm(forms.ModelForm):
+    """ 
+        Using ModelForm to directly convert the CustomUser model into the EditProfileForm form.
+    """
+
+    class Meta:
+        model = CustomUser
+        fields = (
+            'email',
+            'first_name',
+            'last_name',
+            'slack_display_name',
+            'user_type',
```

Comment:
Could you please remove this field? I don't think the user should be able to change this. Instead a change in `user_type` should come from an admin.